tests/load/test_read_interfaces.py

```diff
@@ -314,7 +324,44 @@ def test_read_interfaces_filesystem(destination_config: DestinationTestConfigura
             "read_pipeline", dataset_name="read_test", dev_mode=True, destination=gcp_bucket
         )
         _run_dataset_checks(pipeline, destination_config)
-        assert pipeline.destination_client().config.credentials
+
+
+@pytest.mark.essential
+@pytest.mark.parametrize(
+    "destination_config",
+    destinations_configs(
+        table_format_filesystem_configs=True,
+        with_table_format="delta",
+        bucket_exclude=[SFTP_BUCKET, MEMORY_BUCKET],
+    ),
+    ids=lambda x: x.name,
+)
+def test_delta_tables(destination_config: DestinationTestConfiguration) -> None:
+    os.environ["DATA_WRITER__FILE_MAX_ITEMS"] = "700"
+
+    pipeline = destination_config.setup_pipeline(
+        "read_pipeline",
+        dataset_name="read_test",
+    )
+
+    # in case of gcs we use the s3 compat layer for reading
```

Comment:
this is a bit bad tbh. writing delta tables only works with gcs native, reading them in duckdb requrires the s3 api. this should be investigated at some point by @jorritsandbrink I think.